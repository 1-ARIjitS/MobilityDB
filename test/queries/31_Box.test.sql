/*****************************************************************************
 * Box
 *****************************************************************************/

SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b && t2.b;
SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b @> t2.b;
SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b <@ t2.b;
SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b ~= t2.b;

/*****************************************************************************/