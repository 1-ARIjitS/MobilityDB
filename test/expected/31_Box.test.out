/*****************************************************************************
 * Box
 *****************************************************************************/
SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b && t2.b;
 count 
-------
   236
(1 row)

SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b @> t2.b;
 count 
-------
   104
(1 row)

SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b <@ t2.b;
 count 
-------
   104
(1 row)

SELECT count(*) FROM tbl_box t1, tbl_box t2 where t1.b ~= t2.b;
 count 
-------
   100
(1 row)

/*****************************************************************************/
